etcdctlv3/command/ping_command.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/spf13/cobra"
+	"github.com/coreos/etcd/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/etcd/clientv3"
+)
+
+// NewPingCommand returns the cobra command for "ping".
+func NewPingCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ping",
+		Short: "ping the member specified in `--endpoints` flag",
+		Run:   pingCommandFunc,
+	}
+	return cmd
+}
+
+// pingCommandFunc executes the "ping" command.
+func pingCommandFunc(cmd *cobra.Command, args []string) {
+	endpoints, err := cmd.Flags().GetStringSlice("endpoints")
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+
+	cert, key, cacert := keyAndCertFromCmd(cmd)
+	cfgs := []*clientv3.Config{}
+	for _, ep := range endpoints {
+		cfg, err := newClientCfg([]string{ep}, cert, key, cacert)
+		if err != nil {
+			ExitWithError(ExitBadArgs, err)
+		}
+		cfgs = append(cfgs, cfg)
+	}
+
+	for _, cfg := range cfgs {
```

Comment:
That complicates the implementation since we have to do a list member call first. I want all the command to be as simple as possible first. We can combine more later.